### PR TITLE
Components setup

### PR DESCRIPTION
**Pull request by michaelamponsah**
### **In the branch, the following requirements have been implemented;**

- [x] Initialized a `react app` 
- [x] Built custom components for sections of the application. Components include Navbar, Book cards, Buttons, and forms.
- [x] Used `react-router`  for app navigations.

 
book-store-1

### CODE REVIEW

File: src/components/book/Book.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      <div className={`${styles['stats-details-wrapper']}`}>
+        <div className={`${styles['book-details']}`}>
+          <div className={`${styles['book-details']}`}>
+            <span className={styles.genre}>{genre}</span>
+            <h3 className={`${styles['book-title']}`}>{title}</h3>
+            <p className={`${styles['author-name']}`}>{author}</p>
+          </div>
+          <div className={styles.comments}>
+            Comments | Remove | Edit
+          </div>
```
Comment (PinkMoon25) [inline src/components/book/Book.jsx:22]: - [ ] Great job with the project, you have done really well :clap: , however as mentioned in the project requirements **"Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet)."** 
we are required to render each book with a remove button, so how about changing these from a div to a button :pray:

File: src/index.css [hunk 2]
```diff
@@ -1,13 +1,33 @@
...
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
```
Comment (PinkMoon25) [inline src/index.css:25]: - [ ] [OPTIONAL] If we are not using this stylesheet how about removing this boilerplate generated with CRA.

**Review by PinkMoon25 — CHANGES_REQUESTED** [review]
Hi @michaelamponsah , 

Good job so far! you have done really wel :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
linter checks :heavy_check_mark: 
professional Readme :heavy_check_mark: 
routing :heavy_check_mark: 
gitflow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/book/Book.jsx (hunk 1 above)
Comment (michaelamponsah) [reply src/components/book/Book.jsx:22]: @PinkMoon25, Many thanks for your review! 🚀

Comment (PinkMoon25) [reply src/components/book/Book.jsx]: Pleasure is mine 👍

**Pull request merged**